### ISSUE TITLE
Midi Learn - and forget

### ISSUE THREAD

**Issue by VinceLewis** [amsynth/amsynth#101]
### Expected behavior
I am expecting that when I assign controllers from my midi device to AMSYNTH, then it remembers them

### Actual behavior
one time it remembered it, the next time not. (Steps below)

### Information

  - Which version of amsynth are you using?
amsynth 1.5.1

  - Which build of amsynth are you using? (stand-alone application, LV2, DSSI, VST)
Stand alone

  - If using amsynth as a plugin, which host application are you using?
N/A

  - Which operating system & version are you using?
Ubuntu Studio 14.04

### Steps to reproduce the behavior

- I opened AMSYNTH
- I assigned some midi controllers from my keyboard.
- I closed AMsynth
- I re-opened it to see if it would remember the controllers. It did.
- I deliberately started a second instance of AMSYNTH on another midi port, as I wanted to see if I could control 2 instances from one keyboard (my keyboard allows for selecting midi channel). 
- I did NOT close down the first instance before starting the second
- The controller map reset itself.

**Comment by nixxcode** [comment]
I am not able to reproduce this issue after following the reported steps.

Does it happen repeatedly on your end?

**Comment by danboid** [comment]
I may be seeing the same bug here. I'll describe how I see it running amsynth git under TrueOS.

When I right-click a dial under amsynth and choose MIDI Learn, the 'MIDI controller' dialog always wants to revert to '13 Effect Control 2'. To assign any other controller, I have to move the desired controller and then click 'OK' in the midi learn window as fast as I can so that it doesn't revert back to '13 Effect Control 2' but maybe its MIDI controller playing up rather than a fault of amsynth or maybe my prob is something unrelated to the original issue?

**Comment by danboid** [comment]
Also, if I adjust more than a couple of MIDI controllers in quick succession when using the MIDI learn, it crashes amsynth git standalone. I should prob open another ticket for that as I don't think VinceLewis was seeing that happen?

**Comment by nickdowell** [comment]
Closed due to inactivity / inability to reproduce.
